### DISCUSSION TITLE
Error 500 - Request failed

### DISCUSSION THREAD

**Discussion by marianoka** [Security-Onion-Solutions/securityonion#4927] [General]
Hi So Gurus!
I'm getting this error after installing so-velociraptor. The thing is it works, but suddenly I get this error and even I have tried to go back, still get the soc interface with this error
Thx
Mariano
![imagen](https://user-images.githubusercontent.com/9213379/126915519-03042c62-8ca4-4ba4-ac54-20e14a17daa2.png)

**Comment by jertel** [comment]
What does your browser dev tools console report?

**Comment by marianoka** [comment]
Hi jertel
Thanks for your answer. I have found the solution. The problem was the Nginx conf file.
I had to edit based on a clear conf and it works.

**Reply by devstrauss** [reply]
what did you do to the nginx conf file?
